proxy/src/schema.rs

```diff
@@ -44,6 +52,82 @@ impl Into<ProjectId> for IdInput {
     }
 }
 
+#[derive(GraphQLObject)]
+struct Branch {
+    name: String,
+}
+
+#[derive(GraphQLObject)]
+struct Tag {
+    name: String,
+}
+
+#[derive(GraphQLObject)]
+struct Person {
+    name: String,
+    email: String,
+}
+
+#[derive(GraphQLObject)]
+struct Commit {
+    sha1: String,
+    author: Person,
+    summary: String,
+    message: String,
+    time: String,
+}
+
+// FIXME(xla): This should be a `std::convert::TryFrom` and needs to be addressed together with
+//             consistent error handling.
+impl From<&radicle_surf::vcs::git::git2::Commit<'_>> for Commit {
```

Comment:
I'm thinking that `Commit` being exposed from `radicle-surf` due to it being the `A` in our model justifies it being re-exported. I think it'd be good to start tracking what we would like exposed, and then limiting the re-export

Reply:
@FintanH Do you have a preferred way of tracking these?

Reply:
I think an issue with a checklist would be good. I think at the moment want:

- Commit
- BranchType
- Oid

Think that may be it for now.

Reply:
Something lke: radicle-dev/radicle-surf#41 ?

Reply:
YES muchos love ❤️